lib/Support/Path.cpp

```diff
@@ -47,13 +47,16 @@ CanonicalPathCacheImpl::getCanonicalPath(StringRef Path, StringRef WorkingDir) {
     return CanonicalFilePath();
 
   SmallString<256> AbsPath;
+  bool IsRelative = false;
```

Comment:
Just to be clear, none of the Path.cpp changes would be required after the changes to not canonicalize the out file above right?